packages/create-flex-plugin/templates/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "postinstall": "flex-check-start",
```

Comment:
I think `prebuild` is a better hook; just in case

Reply:
Doing it on `postinstall` has the benefit of the file existing before a `build` or `run`.

Reply:
That's fair. At least for actual release builds this wouldn't really matter.